spec/controllers/application_controller/explorer_spec.rb

```diff
@@ -40,7 +40,8 @@
         ems = FactoryGirl.create(:ems_vmware, :ems_folders => [ems_folder])
 
         user = FactoryGirl.create(:user_admin)
-        user.current_group.set_managed_filters([["/managed/service_level/gold"]])
+        user.current_group.entitlement = Entitlement.create!(:filters => {'managed'   => [["/managed/service_level/gold"]],
+                                                                          'belongsto' => []})
```

Comment:
I'd prefer not hardcoding these managed and belongto hash keys.  What do you think of using the entitlement.set_managed_filter setter instead?